api-catalog-ui/frontend/src/components/Wizard/configs/wizard_regex_restrictions.jsx

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/prefer-default-export
+export const wizRegex = {
+    gatewayUrl: { value: '^(/[a-z]+\\/v\\d+)$', tooltip: 'Format: /api/vX, Example: /api/v1' },
```

Comment:
I've tested this too, it doesn't work. The correct one should be:
^(\/[a-z]+\/v\d+)$